temporalio/ui
# Dark mode button

## Description & motivation 💭 
Added new button styles to button component

### Screenshots (if applicable) 📸 

### Design Considerations 🎨 
https://www.figma.com/file/LNN3Cu8Akqb240WmJmbtsS/Holocene?node-id=3789%3A216118&mode=dev
Will confirm that this looks okay with Ash. 

## Testing 🧪 

### How was this tested 👻 
WILL UPDATE TESTS ONCE DESIGN CONFIRMS THESE ARE GOOD TO GO


### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
Please check story book and make sure buttons look like they do in figma designs

## Checklists

### Light
- [ ] primary
- [ ] secondary
- [ ] destructive
- [ ] ghost

### Dark
- [ ] primary
- [ ] secondary
- [ ] destructive
- [ ] ghost


### Draft Checklist 

### Merge Checklist 

### Issue(s) closed 
dark mode buttons

## Docs

### Any docs updates needed?
